modules/dynamic-plugins/ref-community-plugins.template.adoc

```diff
@@ -3,7 +3,14 @@
 
 = Community plugins
 
-{product-short} includes the following %%COUNT_3%% community plugins:
+[IMPORTANT]
+====
+{product} ({product-very-short}) includes a select number of community-supported plugins, available for customers to enable and configure. These community plugins are exported by {company-name} as dynamic plugins, and are provided with support scoped per https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope] terms. 
```

Comment:
Please confirm that the link added here is correct.

Reply:
@Gerry-Forde  and @hmanwani-rh - per above comment, please link to the RHDH support policy page for community support statement. https://access.redhat.com/policy/developerhub-support-policy